docs/validator-guides/add-validator-with-msig.md

```diff
@@ -0,0 +1,258 @@
+| Camino (mainnet):   | https://suite.camino.network/     | Production version: Please use this if you are unsure which version to choose.       |
+| Columbus (testnet): | https://dev.suite.camino.network/ | Development version: Please use this version if you would like to test new features. |
+
+Please be advised that both of these wallets have access to both the mainnet and
+testnet. However, one of them is the production version while the other is
+designated for staging and development purposes.
+
+:::
+
+:::caution VERIFY WALLET ADDRESS
+
+Always verify that the domain is **camino.network** while using the wallet.
+
+![](/img/add-validator/wallet-ssl-check-0.png)
+
+:::
+
+## Overview
+
+To add a validator node using Camino Wallet and a multi-sig wallet, you will need to follow these steps:
+
+1. Activate your Multi-Sig Wallet
+1. Check and verify the requirements
+1. Retrieve your node's private key
+1. Register your node with your wallet
+   - Initiate the Register Node transaction
+   - Collect signatures from other members of the wallet
+   - Execute the transaction
+1. Add your node as a validator
+   - Set the end dates
+   - Initiate the Add Validator transaction
+   - Collect signatures from other members of the wallet
+   - Execute the transaction
+1. Verify that the validator is online
+
+## 1. Activate your Multi-Sig Wallet
+
+Firstly, you will need to [activate your multi-sig wallet](/guides/multisig-wallets#managing-multi-sig-wallets-with-camino-wallet).
+To do this, log in to your wallet using your key phrase and navigate to the Manage Keys
+section (or click the switch button at the top right).
+
+If you are a member of a multi-sig wallet, an import wallets dialog box will appear. Click the
+Import Wallets button to pull in the multi-sig wallets that you are a member of.
+
+Next, click the small star icon on the right side of the multi-sig address to activate it.
+
+For more detailed information, please refer to the [Multi-Sig Wallets & Aliases](/guides/multisig-wallets) documentation.
+
+## 2. Requirements
+
+Before you begin the process of adding a validator node, there are a few requirements you must fulfill.
+Below is a list of these requirements for the mainnet (`camino`) and testnet (`columbus`).
+
+### Camino Mainnet & Columbus Testnet
+
+- **NodeID**: This is the address of your node, similar to a regular wallet address.
+- **Node's PrivateKey**: You will need this to prove ownership of the **NodeID**.<br/>
+  Keep it safe and secure!
+- **100,000 non-bonded CAMs** in your P-Chain wallet. And a few CAMs for paying for transaction fees.
+- **Consortium Member**: Your wallet address must be a consortium member.<br/>
+  If you are unsure about this, please reach out to us on [Discord](https://discord.gg/camino) for clarification. If you participated in the pre-sale, you are already a Consortium Member.
+- **KYC/KYB Verified**: You must be Know-Your-Customer (KYC) or Know-Your-Business (KYB) verified.
```

Comment:
For now KYB verification is not enabled, so might be worth not mention it for now.

Reply:
Because we use "KYB" in the wallet UI, I believe it's better to keep it here also.